gips/0083.md

```diff
@@ -123,11 +157,20 @@ Having no queries like subgraphs have, Proofs of Indexing for Substreams are alw
 
 Substreams endpoints provide signed attestations with each block's response, as per [the `attestation` field](https://github.com/streamingfast/substreams/blob/develop/proto/sf/substreams/rpc/v2/service.proto#L96) in the `BlockScopedData` response, matching the operator key specified in [the `attestation_public_key` field](https://github.com/streamingfast/substreams/blob/develop/proto/sf/substreams/rpc/v2/service.proto#L106) of the `SessionInit` response.
 
-The attestation payload MUST be signed by a special _attestation key_, derived from the operator's key, so that disputes can be opened on a separate allocation destined for disputes.
+The payload signed over is composed of:
+- the Substreams output module's payload, hashed with SHA256 (32 bytes), followed by:
+- the 'b' character, followed by:
+- the block ID as UTF-8 encoded string, followed by:
+- the 'm' character, followed by:
+- the module hash for the top-level Substreams module being attested (32 bytes)
+
+The attestation payload MUST be signed by a special _attestation key_, derived from the operator's key, so that disputes can be opened on the _Substreams Service Deployment ID_.
```

Comment:
The key is the private key corresponding to the allocationID (that is an address)